doc/tutorials/introduction/building_tegra_cuda/building_tegra_cuda.markdown

```diff
@@ -0,0 +1,620 @@
+Building OpenCV for Tegra with CUDA {#tutorial_building_tegra_cuda}
+===================================
+
+This document is a basic guide to building the OpenCV libraries with CUDA support for use in the Tegra environment. It covers the basic elements of building the version 3.1.0 libraries from source code for three (3) different types of platforms:
+
+* NVIDIA DRIVE&trade; PX 2 (V4L)
+* NVIDIA<sup>&reg;</sup> Tegra<sup>&reg;</sup> Linux Driver Package (L4T)
+* Desktop Linux (Ubuntu 14.04 LTS and 16.04 LTS)
+
+This document is not an exhaustive guide to all of the options available when building OpenCV. Specifically, it covers the basic options used when building each platform but does not cover any options that are not needed (or are unchanged from their default values). Additionally, the installation of the CUDA toolkit is not covered here.
+
+This document is focused on building the 3.1.0 version of OpenCV, but the guidelines here may also work for building from the master branch of the git repository. There are differences in some of the CMake options for builds of the 2.4.13 version of OpenCV, which are summarized below in the @ref building_opencv_24X section.
+
+Most of the configuration commands are based on the system having CUDA 8.0 installed. In the case of the Jetson TK1, an older CUDA is used because 8.0 is not supported for that platform. These instructions may also work with older versions of CUDA, but are only tested with 8.0.
+
+Native Compilation vs. Cross-Compilation
+----------------------------------------
```

Comment:
Lowered to ###, slight change to the heading to clarify that this is only a side-note.